### DISCUSSION TITLE
How to turn off mouse mode?

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#829] [Q&A] [answered]
I want to use system to paste,Now 
The existing system is not compatible with mine, I want to turn off mouse mode

**Comment by ayamir** [comment, marked as answer]
add `mouse = false,` above this line:
https://github.com/ayamir/nvimdots/blob/daf1b3dba71fbc44b857505521f8ebdf4f81ed7d/lua/core/options.lua#L53
